Projects/Features/GuestList/Sources/GuestListInteractor.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  GuestListInteractor.swift
+//  GuestList
+//
+//  Created by Gunoo on 2022/07/09.
+//  Copyright (c) 2022 kr.mash-up. All rights reserved.
+//
+//  This file was generated by the Clean Swift Xcode Templates so
+//  you can apply clean architecture to your iOS and Mac projects,
+//  see http://clean-swift.com
+//
+
+import UIKit
+import Models
```

Comment:
P3
> 임포트 규칙
모듈 임포트는 알파벳 순으로 정렬합니다. 내장 프레임워크를 먼저 임포트하고, 빈 줄로 구분하여 서드파티 프레임워크를 임포트합니다.

[스타일쉐어 Coding Convention](https://github.com/StyleShare/swift-style-guide#네이밍)에 위와 같이 나와있어서 컨벤션을 지킨다면 아래와 같이 사용되어야 할 것 같습니다.
```suggestion
import Models
import UIKit
```

Reply:
Lint 설정이 되어있으면 바로 알아챌수 있지만 저희가 Lint 세팅이 안되어 있어서 ㅋㅋㅋ 알아채기가 쉽지 않네요.. 이거는 그냥 자율적으로 하면 어떨까요?

Reply:
네네 좋습니다.